person.rb

```diff
@@ -1,28 +1,35 @@
-require_relative 'nameable'
+require './nameable'
+require './decorator'
+require './rental'
 
 class Person < Nameable
-  attr_accessor :name, :age
   attr_reader :id
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
-    super()
-    @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super()
   end
 
-  def correct_name
-    @name
-  end
+  private
 
   def of_age?
     @age >= 18
   end
 
-  private
+  public
 
   def can_use_services?
     of_age? || @parent_permission
   end
+
+  def correct_name
+    @name
+  end
+
+  def add_rentals(date, _person, book)
+    Rental.new(date, self, book)
+  end
```

Comment:
- [ ] There is no need for the parameter `_book` since it's not used in the method.